### ISSUE TITLE
Not_Found error when assign a constant in discrete init

### ISSUE THREAD

**Issue by lbinria** [imitator-model-checker/imitator#56]
## Description

A Not_Found error is raised when assign a constant in a discrete init

```
var 
    x,
        : clock;
    i = 1, 
        : int;
    p
        : parameter;

init := {

    discrete = 
        i := 1
    ;
    
}

## Solution

I just had to check that variable is a constant or variable. In case of constant, it raise an `InvalidExpression` exception with an appropriate message.

**Comment by lbinria** [comment]
Bug fixed :
e0bc8768aa1ad1b2d9d58b126d0f94b7cbf4cd7a
